setup.py

```diff
@@ -34,8 +32,10 @@ def find_version(version_file_path):
 extensions = []
 cmdclass = {}
 
-force_cuda = os.getenv("FORCE_CUDA", "0") == "1"
-if (torch.cuda.is_available() and CUDA_HOME is not None) or force_cuda:
+build_cuda_extensions = os.getenv("BUILD_CUDA_EXTENSIONS", "0") == "1"
+if build_cuda_extensions:
+    from torch.utils.cpp_extension import BuildExtension, CUDAExtension
+
```

Comment:
Thank you for the change! Should we modify the warning statement in the else: clause?

does it make sense to still check for cuda and CUDA_HOME?  Looks like we will require users to set the environment var: BUILD_CUDA_EXTENSIONS if they want to use fused_adam. It's not being used right now so maybe its ok to leave it as is?

Reply:
Also, do we need to modify the changelog?

Reply:
> Thank you for the change! Should we modify the warning statement in the else: clause?
> 
> does it make sense to still check for cuda and CUDA_HOME? Looks like we will require users to set the environment var: BUILD_CUDA_EXTENSIONS if they want to use fused_adam. It's not being used right now so maybe its ok to leave it as is?

It feels simpler just to fail if the user sets BUILD_CUDA_EXTENSIONS but then does not have CUDA_HOME set.